src/os/bluestore/aio.h

```diff
@@ -16,12 +16,12 @@ struct aio_t {
   int fd;
   boost::container::small_vector<iovec,4> iov;
   uint64_t offset, length;
-  int rval;
+  uint64_t rval;
```

Comment:
i think we should be using a signed type for rval. as it could represent a failure of async i/o. so `long` would be better? it also matches the type used by libaio. please note, Ceph could be running a 32bit CPU.